GVRf/Framework/framework/src/main/java/org/gearvrf/io/GVRCursorController.java

```diff
@@ -988,7 +1002,7 @@ public boolean isEventHandledBySensorManager() {
         return eventHandledBySensor;
     }
 
-    void setEventHandledBySensor()
+    public void setEventHandledBySensor()
```

Comment:
setEventHandledBySensor is an example why I am against having the io package. As long as we need such methods to be public, we should keep the io stuff in the main package.